### PR TITLE
multiplicity-Pb-Pb adding filtering for Mc generated by charges

### PR DESCRIPTION
**Pull request by hhesouno**

### CODE REVIEW

File: PWGMM/Mult/Tasks/CMakeLists.txt [hunk 1]
```diff
@@ -60,6 +60,8 @@ o2physics_add_dpl_workflow(heavy-ion-mult
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
                     COMPONENT_NAME Analysis)
 
+
+
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/CMakeLists.txt:64]: Please remove extra lines.

File: PWGMM/Mult/Tasks/multiplicityPbPb.cxx [hunk 2]
```diff
@@ -155,6 +155,11 @@ struct multiplicityPbPb {
     for (auto& mcParticle : mcParticles) {
       ++MCparticleCounter;
       if (mcParticle.isPhysicalPrimary()) {
+        auto pdgparticle = pdg->GetParticle(mcParticle.pdgCode());
+        if (pdgparticle != nullptr)
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/multiplicityPbPb.cxx:159]: The coding guidelines require braces even around single-line branches.

File: PWGMM/Mult/Tasks/multiplicityPbPb.cxx [hunk 3]
```diff
@@ -155,6 +155,11 @@ struct multiplicityPbPb {
...
+          if (pdgparticle->Charge() == 0)
```
Comment (aalkin) [inline PWGMM/Mult/Tasks/multiplicityPbPb.cxx:160]: This value is stored as a floating point number, comparison of a floating point number to 0 is not guaranteed to be always correct. You can do something like `std::abs(pdgparticle->Charge()) > 2.` (note that the charge value is in units of |e|/3 and simulated particle stack may contain quarks).

**Review by aalkin — CHANGES_REQUESTED** [review]

File: PWGMM/Mult/Tasks/multiplicityPbPb.cxx (hunk 3 above)
Comment (hhesouno) [reply PWGMM/Mult/Tasks/multiplicityPbPb.cxx:160]: Hello, 
thank you, I need to distinguish charged and neutral particles. Will it work if I use std::abs(pdgparticle->Charge()) > 1?

Comment (aalkin) [reply PWGMM/Mult/Tasks/multiplicityPbPb.cxx:160]: The value of `1` corresponds to 1/3 of the electron charge, so using condition `>1` would let particles with the charge of 2/3 of the electron charge into your sample. Asking for a particles with |charge| more than 2, i.e. `> 2/3|e|`, or, alternatively, skipping particles with |charge| less than 3. `< 1|e|`, is what you want.

**Pull request closed without merging**